app/json/slate_lineup_configs.json

```diff
@@ -35,29 +33,13 @@
   },
   {
     "id": "e0c0cf7e-3a02-4a2f-a614-5a0cad4e3273",
-    "description": "Android Discover Tab Personalized Algorithmic Topic",
+    "description": "Android Discover Tab Algorithmic Topic",
     "experiments": [
       {
         "description": "default layout",
-        "rankers": [
-          "top1-topics"
-        ],
+        "rankers": [],
         "slates": [
-          "d9c6ddfa-a58d-402d-9664-4190ba197ea6",
-          "e6c8db8d-cae7-4947-a57e-872dec639472",
-          "8aaac382-9c27-42ab-9470-ac1dfc666262",
-          "ce96db55-f32e-48f1-929f-216eb2e8c082",
-          "884414d5-502d-4e1a-8e70-1dc8eb4a345a",
-          "44dff273-3604-40e9-a0b5-bf3852581990",
-          "494eaa68-f048-4f52-bb9a-5e7d2526f5cb",
-          "e6600ffe-b65a-42d3-b570-f460f3d7326e",
-          "72ce0827-9dbc-470f-9722-39476daaeb0f",
-          "9901851d-ea76-4667-8dc7-0c1677ecb910",
           "ab6048ca-5c61-4e87-aefd-7a612b60ab7b",
```

Comment:
Kept science slate based on [popularity of topic pages](https://app.mode.com/getpocket/reports/8db2aee53ddf).

Reply:
there is a fallback lineup available as well.  I don't think this has large impact either way